### ISSUE TITLE
Read globals (`$_GET`, `$_POST`, `$_FILES`, `$_REQUEST`, etc) from `http.Request` types

### ISSUE THREAD

**Issue by deuill** [deuill/go-php#51]
There is currently no easy way of populating request-specific globals in PHP contexts (apart from binding variables and setting globals in PHP). For `go-php` to be useful in replicating or replacing the standard `apache`/`nginx`/`php-fpm` stack, we need to be able to handle requests transparently.

One possible implementation is having `Context` conform to the `http.Handler` interface, therefore allowing for transparent handling of incoming requests.

**Comment by PaulBGD** [comment]
Has there been any work on this? Would be very useful for me.

**Comment by deuill** [comment]
Not really, the library is pretty much unmaintained though. Feel free to contribute if you want.
